source/dub/compilers/utils.d

```diff
@@ -117,6 +117,18 @@ unittest {
 }
 
 
+package void addDynamicLibName(ref BuildSettings settings, in BuildPlatform platform, string fileName)
```

Comment:
Can you add a command explaining why this function is needed so that the next guy doesn't have to `git blame` ?
Don't need to do the whole `Params` thing but a simple description is useful.

Reply:
Will do.

Reply:
Added.